airflow/contrib/hooks/redis_hook.py

```diff
@@ -66,3 +66,47 @@ def get_conn(self):
                 db=self.db)
 
         return self.redis
+
+    def set_key_value(self, k, v):
+        """
+        Create redis key ${k} with value ${v}
+        """
+        client = self.get_conn()
+
+        return client.set(k, v)
+
+    def get_key(self, k):
+        """ Get value of key ${key} stored in Redis """
+        client = self.get_conn()
+
+        return client.get(k)
+
+    def delete_key(self, k):
+        """ Delete key ${k} from redis """
+        client = self.get_conn()
+
+        return client.delete(k)
+
+    def publish_message(self, channel, message):
+        """ Publish message ${message} to channel ${channel}"""
+        client = self.get_conn()
+
+        client.publish(channel=channel, message=message)
```

Comment:
Does this method returns True/False? If yes it would be great to return this information from hook method.

Reply:
returns int - number of subscribes that received the message. so will return that ;)